server/channels/app/platform/web_conn.go

```diff
@@ -776,6 +777,15 @@ func (wc *WebConn) createHelloMessage() *model.WebSocketEvent {
 		wc.Platform.ClientConfigHash(),
 		ee))
 	msg.Add("connection_id", wc.connectionID.Load())
+
+	hostname, err := os.Hostname()
+	if err != nil {
+		wc.Platform.logger.Error("Could not get hostname", mlog.Err(err))
```

Comment:
Apologies for the drive-by review, but just wanted to quickly highlight a few minor things:
1. We usually choose the WARN log level if the server is able to gracefully handle the failure, and there's no major loss of functionality. In this case, it seems like the WARN level would be the right choice: https://developers.mattermost.com/contribute/more-info/server/style-guide/#log-levels
2. We should add as much context as possible to an error log line. Here, I would expect the user_id and connection_id to be logged as well. This lets the admin using a log management software (like Loki/Splunk) easily apply label filters, and aggregate such events by those filters.